services/QuillLMS/app/models/concerns/user_flagset.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+# This concern is only intended to be included in the User model
+module UserFlagset
+  extend ActiveSupport::Concern
+
+
+  included do
+    FLAGSETS = {
+
+      production: {
+        display_name: 'production',
+        flags: {
+          Flags::PRODUCTION =>        { display_name: 'Production' }
+        }
+      },
+
+      alpha: {
+        display_name: 'Alpha',
+        flags: {
+          Flags::ALPHA =>             { display_name: 'Alpha' },
+          Flags::EVIDENCE_BETA1 =>    { display_name: 'Evidence Beta 1' },
+          Flags::EVIDENCE_BETA2 =>    { display_name: 'Evidence Beta 2' },
+          Flags::BETA =>              { display_name: 'Beta' },
+          Flags::COLLEGE_BOARD =>     { display_name: 'College Board' },
+          Flags::PRODUCTION =>        { display_name: 'Production' }
+        }
+      },
+
+      evidence_beta1: {
+        display_name: 'Evidence Beta 1',
+        flags: {
+          Flags::EVIDENCE_BETA1 =>    { display_name: 'Evidence Beta 1' },
+          Flags::EVIDENCE_BETA2 =>    { display_name: 'Evidence Beta 2' },
+          Flags::PRODUCTION =>        { display_name: 'Production' }
+        }
+      },
+
+      evidence_beta2: {
+        display_name: 'Evidence Beta 2',
+        flags: {
+          Flags::EVIDENCE_BETA1 =>    { display_name: 'Evidence Beta 1' },
+          Flags::PRODUCTION =>        { display_name: 'Production' }
+        }
+      },
```

Comment:
This one is confusingly-named if it's accurate.  The flagset is called `beta2`, but it gives access to `beta1`?  We either need a better name, or this may be supposed to give access to `beta2`?

Reply:
Address, thanks.